packages/frontend/src/utils/activity/getTransactionMonthlyCount.ts

```diff
@@ -0,0 +1,11 @@
+import { ActivityApiChartPoint } from '@l2beat/types'
+
+export function getTransactionMonthlyCount(
```

Comment:
I thought we already had this?

Reply:
Renamed the file but forgot to delete the old one. Now updated, so `getTransactionCount` can count both monthly and weekly